### PR TITLE
bump pretender and disable request tracking in pretender

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

File: addon/server.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ function createPretender(server) {
          mirage/config.js file. Did you forget to add your namespace?`
       );
     };
-  });
+  }, { trackRequests: false });
```
Comment (kellyselden) [inline addon/server.js]: What is the affect of this on users? Also, is this testable?

File: package.json [hunk 2]
```diff
@@ -80,4 +80,4 @@
   "ember-addon": {
     "configPath": "tests/dummy/config"
   }
-}
+}
```
Comment (kellyselden) [inline package.json]: Can you revert this please, as npm will add this back the next time it gets modified.

File: addon/server.js (hunk 1 above)
Comment (bekzod) [inline addon/server.js]: no effect to use I think, originally solves this https://github.com/samselikoff/ember-cli-mirage/issues/509

**Comment by samselikoff** [comment]
@Turbo87 @kellyselden thank yall so much for your recent help with these PRs, can't tell you how nice it is to find this stuff getting merged without any intervention on my part! ❤️

**Pull request merged**